### PR TITLE
enable 'discard=on,ssd=1' for all volumes (space saving measure)

### PR DESCRIPTION
**Pull request by PC-Admin**
Applying the latest space saving settings to every host. :+1:

### CODE REVIEW

**Review by Zorlin — APPROVED** [review]
LGTM if it's been successfully tested on dev-demo

**Comment by PC-Admin** [comment]
These changes are also found in another PR: https://github.com/application-research/ehi-proxmaas/pull/18

After one last round of testing I'll be merging that PR instead for simplicity.

**Pull request closed without merging**
